spec/isPalindrome.js

```diff
@@ -9,23 +9,23 @@ import isPalindrome from '../src/isPalindrome'
 describe('isPalindrome', function(){
   context('when given "radar"', function(){
```

Comment:
Why choose to set up a separate context here? It's unnecessary - especially since you're writing only one test that doesn't share context with any of the other tests you're writing.